GREEDY/ImagePreparation/Deskew.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GREEDY.ImagePreparation
+{
+    class Deskew
```

Comment:
Don't know if it is just me but the class name gives no idea what does it do. Also shouldn't it be a public class?